docs/ui/prepare-migration/dataflow-tuning.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Tune Dataflow (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Tune Dataflow (Optional)
+{: .no_toc }
+
```

Comment:
Let's an `{ important }` tag that the top saying "This configuration is relevant especially you want to run Dataflow inside a VPC."